agents/postgres/pgstatstatements/pgstatstatements.go

```diff
@@ -207,19 +202,17 @@ func makeBuckets(q *reform.Querier, current, prev map[int64]*pgStatStatementsExt
 		default:
 			l.Debugf("Normal query: %s.", currentPSS)
 		}
-		currentPSS.Database = getDatabaseName(currentPSS.DBID, prevPSS, q, l)
-		currentPSS.Username = getUserName(currentPSS.UserID, prevPSS, q, l)
-		currentPSS.Tables = getTables(currentPSS.Query, prevPSS, l)
 
 		mb := &agentpb.MetricsBucket{
 			Common: &agentpb.MetricsBucket_Common{
-				Database:    pointer.GetString(currentPSS.Database),
+				Database:    currentPSS.Database,
 				Tables:      currentPSS.Tables,
-				Username:    pointer.GetString(currentPSS.Username),
+				Username:    currentPSS.Username,
 				Queryid:     strconv.FormatInt(currentPSS.QueryID, 10),
 				Fingerprint: currentPSS.Query,
 				NumQueries:  count,
 				AgentType:   inventorypb.AgentType_QAN_POSTGRESQL_PGSTATEMENTS_AGENT,
+				IsTruncated: currentPSS.QueryTruncated,
```

Comment:
@askomorokhov Docs say:
> Indicates if query examples is too long and was truncated.

But we truncate fingerprint, not an example. Will it work? Is qan-api2 even actually uses this field?

Reply:
Or we should send `ExampleFormat_FINGERPRINT` – something we don't do anywhere?